AFQ/tasks/segmentation.py

```diff
@@ -61,6 +63,18 @@ def segment(data_imap, mapping_imap,
         trx = load_trx(streamlines, data_imap["dwi"])
         trx.streamlines._data = trx.streamlines._data.astype(np.float32)
         tg = trx.to_sft()
+    elif streamlines.endswith(".tck.gz"):
+        # uncompress tck.gz to a temporary tck:
+        temp_tck = streamlines.replace(".gz", "")
```

Comment:
I would do something like: 

```suggestion
        temp_tck = os.join(makedtmp(), streamlines.replace(".gz", ""))
```

You'll need to also add: 

```
import os.path as op
from tempfile import mkdtemp()
```
At the top of the file. 

This will create a temporary directory, save the file in there, allow you to read that in and use it, and then get rid of the file and the directory when they are no longer used.